Valet.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		AC2C2D481B8CE184008F4EE8 /* Valet_iOS_Framework.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = AC2C2D311B8CE184008F4EE8 /* Valet_iOS_Framework.framework */; };
```

Comment:
What do you think about making `Valet.framework` should be the output name of the framework here? This file will be nested inside of `Carthage/Builds/iOS` anyway, so the additional naming of iOS is redundant, and `_Framework.framework` is also redundant.